Make complexity requirements opt in (#38)
fusionspim/php-password-checker: README.md

$checker->validate('clemfandango'); // throws PasswordException
$checker->validate('fandango123'); // throws PasswordException
```

### Complexity requirements

Complexity requirements can be enabled to require user passwords to contain a lower case letter, upper case letter, number and special character.

> This is disabled by defaults as it is [not necessarily recommended](https://www.ncsc.gov.uk/collection/passwords/updating-your-approach).

Reviewer: ```suggestion
This is disabled by default, since it [isn't a recommended approach](https://www.ncsc.gov.uk/collection/passwords/updating-your-approach).
```
